mangaki/mangaki/utils/knn.py

```diff
@@ -29,47 +34,22 @@ def set_parameters(self, nb_users, nb_works):
         self.nb_users = nb_users
         self.nb_works = nb_works
 
-    def get_similarity(self, my_user_id, user_id):
-        score = 0
-        for work_id in self.rated_works & set(self.ratings[user_id].keys()):
-            score += self.ratings[my_user_id][work_id] * self.ratings[user_id][work_id]
-        return score
-
-    def get_similarity2(self, my_user_id, user_id):
-        score = self.get_similarity(my_user_id, user_id)
-        my_norm = sqrt(self.get_similarity(my_user_id, my_user_id))
-        their_norm = sqrt(self.get_similarity(user_id, user_id))
-        if my_norm and their_norm:
-            score /= (my_norm * their_norm)
-        return score
-
-    def get_neighbors(self, my_user_id, normalized=False):
-        if normalized:
-            similarity_f = self.get_similarity2
+    def get_neighbors(self, user_ids=None):
```

Comment:
Can we rename this into something like `compute_neighbors`? It doesn't return anything.